model/model.js

```diff
@@ -20,35 +38,30 @@ function getArticleId(articleId) {
     });
 }
 
-function getArticles() {
-  return db
-    .query(
-      `SELECT articles.article_id,
-    articles.title,
-    articles.topic,
-    articles.author,
-    articles.created_at,
-    articles.votes,
-    articles.article_img_url,
-    COUNT(comments.comment_id) AS comment_count
-    FROM articles
-    LEFT OUTER JOIN comments
-   ON articles.article_id = comments.article_id
-   GROUP BY articles.article_id
-   ORDER BY articles.created_at DESC;
-   `
-    )
-    .then(({ rows }) => {
+function getArticles(topic) {
+  if (topic) {
+    return db.query(QueryStringFront+topicQueryString+QueryStringBack, [topic]).then(({ rows }) => {
```

Comment:
We could make this a little more DRY. By just making the one db query and conditionally adding the topicQueryString to the entire query string if the topic variable is defined.